### PR TITLE
added pr template

### LINKED ISSUE #9: Creation of pr template
Can I add a pr template??

### PR DESCRIPTION
**Pull request by Sloth-Panda**
Fixes #9.

sir @praveenscience , let me know if you want any changes on it

### CODE REVIEW

File: Code_of_Conduct.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributor Code of Conduct
```
Comment (praveenscience) [inline Code_of_Conduct.md:1]: Why is this file here?

File: .github/pr_issue_template.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (praveenscience) [inline .github/pr_issue_template.md:63]: Why so many empty lines?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Questions, please answer. 😊

File: Code_of_Conduct.md (hunk 1 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md:1]: Ok will remove

File: .github/pr_issue_template.md (hunk 2 above)
Comment (Sloth-Panda) [reply .github/pr_issue_template.md:63]: I don't know I actually checked for it before making a pr dont know why they are appearing will be checking soon sir

Comment (praveenscience) [reply .github/pr_issue_template.md:63]: Could be just a glitch. Don't worry, just remove and send a commit! 😁

File: Code_of_Conduct.md (hunk 1 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md:1]: sir this contributor code of conduct is not at all there from where you checked that

Comment (praveenscience) [reply Code_of_Conduct.md:1]: @Sloth-Panda As your old PR is not merged yet, it shows it here. My mentors are inactive. Gets me into trouble! 😢

File: .github/pr_issue_template.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+## Related Issue
```
Comment (praveenscience) [inline .github/pr_issue_template.md:1]: Can you add a space after this line, please?

File: .github/pr_issue_template.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Closes: #[issue number that will be closed through this PR]
+
+#### Describe the changes you've made
+
+Give a clear description what modifications you have made
+
+## Type of change
+
+What sort of change have you made :
```
Comment (praveenscience) [inline .github/pr_issue_template.md:12]: Remove space before `:`.

File: .github/pr_issue_template.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+<!--
+Example how to mark a checkbox :-
```
Comment (praveenscience) [inline .github/pr_issue_template.md:14]: Remove space before `:-`.

File: .github/pr_issue_template.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Describe how have you verified the changes made
+
+## Checklist:
+<!--
+Example how to mark a checkbox :-
```
Comment (praveenscience) [inline .github/pr_issue_template.md:29]: Remove space before `:-`.

File: .github/pr_issue_template.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the guidelines of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly whereever it was hard to understand.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] My changes generate no new warnings.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] Any dependent changes have been merged and published in downstream modules.
```
Comment (praveenscience) [inline .github/pr_issue_template.md:38]: Just telling, when you were to accept all these, this you haven't done! Hehe. Do you see? Dependent changes merged? Your dependent change of Contributor Code of Conduct didn't get merged! 😞  So you have to technically uncheck this one! LoL.

File: .github/pr_issue_template.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+## Screenshots
+
```
Comment (praveenscience) [inline .github/pr_issue_template.md:41]: Add a line maybe, saying if this section doesn't apply, please remove.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Just a very few more changes and you're good to go. Yeah, remove the other file please.

File: .github/pr_issue_template.md (hunk 3 above)
Comment (Sloth-Panda) [reply .github/pr_issue_template.md:1]: sure

File: .github/pr_issue_template.md (hunk 7 above)
Comment (Sloth-Panda) [reply .github/pr_issue_template.md:38]: But as it is not yet merged it wont show up show are you asking me to remove this part for now?

**Comment by Sloth-Panda** [comment]
@praveenscience kindly add the relevant tags to this pr


@Salonii02  and @manudd1011  can you guys please review this pr its been few days now

File: .github/pr_issue_template.md (hunk 7 above)
Comment (praveenscience) [reply .github/pr_issue_template.md]: I was just telling you in a funny tone man... 😜

Comment (praveenscience) [reply .github/pr_issue_template.md]: Where's the line now? Did you remove it? 🤔

**Comment by praveenscience** [comment]
Still not completed man! You haven't addressed all the changes, but more than that you have removed something that's not supposed to be removed. Why?

File: .github/pr_issue_template.md [hunk 9]
```diff
@@ -35,7 +36,6 @@ Example how to mark a checkbox :-
 - [ ] I have made corresponding changes to the documentation.
 - [ ] My changes generate no new warnings.
 - [ ] I have added tests that prove my fix is effective or that my feature works.
-- [ ] Any dependent changes have been merged and published in downstream modules.
```
Comment (praveenscience) [inline .github/pr_issue_template.md]: Why did you remove this @Sloth-Panda?

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: .github/pr_issue_template.md (hunk 9 above)
Comment (Sloth-Panda) [reply .github/pr_issue_template.md]: sir, I was confused about what you told so removed that I will be adding that

**Comment by praveenscience** [comment]
Can we have the updates done here, please? 😁

File: .github/pr_issue_template.md (hunk 9 above)
Comment (praveenscience) [reply .github/pr_issue_template.md]: @Sloth-Panda Please put the line back. I was saying your PR was not following that (when "you" wrote the rules, "you" weren't following them for your PR, that's what I meant). Is it clear now? 😅 Sorry for any confusion caused.

Comment (Sloth-Panda) [reply .github/pr_issue_template.md]: Yes, I did get that

**Comment by Sloth-Panda** [comment]
@praveenscience can you please add the tags to it

File: Code_of_Conduct.md (hunk 1 above)
Comment (praveenscience) [reply Code_of_Conduct.md:1]: @Sloth-Panda Why is this file still here?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Just remove the other file and we're good to go. 😁

**Comment by Sloth-Panda** [comment]
@praveenscience It's not there I have checked why it is showing though?

**Comment by praveenscience** [comment]
Interesting... Let's see... Can you go to the branch and then try to delete the file and add it as a commit?

**Comment by Sloth-Panda** [comment]
Okay

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+- [ ] I have made corresponding changes to the documentation.
+- [ ] My changes generate no new warnings.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] Any dependent changes have been merged and published in downstream modules.
+
+## Screenshots
+
+ Original           | Updated
+ :--------------------: |:--------------------:
+ **original screenshot**  | <b>updated screenshot </b> |
```
Comment (praveenscience) [inline .github/pull_request_template.md:45]: Just can you get rid of `<b>` and `</b>`? Just use markdown please.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Looks good to me! 👍  Just one change please?

**Review by praveenscience — APPROVED** [review]
Awesome man. Thanks. 👍

**Review by Salonii02 — APPROVED** [review]

**Comment by Sloth-Panda** [comment]
@praveenscience can you kindly add the tags I am part of both DWOC  and SWOC

**Review by manudd1011 — APPROVED** [review]
